GoogleForCreators/web-stories-wp
# Makes retrieving media from local api bypass preloaded cache when WP Upload modal closed.

## Summary

the preloader is intercepting the request `web-stories/v1/media?context=edit&per_page=100&page=1&_web_stories_envelope=true`
Instead of using the request that is cached we should modify it by appending cache_bust=true when called via after upload media

## Relevant Technical Choices
using a cache_bust parameter for the api call from resetWithFetch (triggered onClose for the WP Media modal).



## User-facing changes

Local media pane refreshes properly when new media is uploaded via the wp media modal. (see #4109)

## Testing Instructions

Upload via local media pane "upload button". Close the modal. The pane should have the new media displayed. 

---



Fixes #4109
